dll/win32/uxtheme/png.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * PROJECT:     ReactOS uxtheme.dll
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     PNG image support in visual styles
+ * COPYRIGHT:   Copyright 2023 Ethan Rodensky <Splitwirez@gmail.com>
+ */
+
+#include "uxthemep.h"
+
+#ifdef MSSTYLES_ACCEPT_PNG_IMAGES
+#include <ole2.h>
+#include <objidl.h>
+#include <gdiplus.h>
+#include <gdipluscolor.h>
+
+BOOL MSSTYLES_TryLoadPng(HINSTANCE hTheme, LPCWSTR szFile, LPCWSTR type, OUT HBITMAP *pngBitmap)
+{
+    HRSRC pngRes = FindResourceW(hTheme, szFile, type);
+    DWORD pngSize = SizeofResource(hTheme, pngRes);
+
+    HGLOBAL pngAlloc = GlobalAlloc(GMEM_MOVEABLE, pngSize);
+    if (!pngAlloc)
+        return FALSE;
+    
+    
```

Comment:
(Superfluous newline).
Also, there is trailing whitespace in this file. Please find and remove it. (notepad++ with show-whitespace option helps viewing that, or VS / VSCode with an extension, etc.)